packages/graphql/lib/util.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var path = require('path');
+
+var hopsConfig = require('hops-config');
+
+exports.getFragmentsFile = function getFragmentsFile () {
+  return path.join(hopsConfig.appDir, 'fragmentTypes.json');
```

Comment:
Why not put this file in `hopConfig.cacheDir` or `hopsConfig.buildDir`?

Reply:
Both of those are being purged regularly - and this file is meant to be added to version control.